<a href="https://colab.research.google.com/github/tareq-stonybrook/datasci_1_loading/blob/main/data/excel_loading1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import pandas as pd

db1_excel=pd.read_excel('https://archive.ics.uci.edu/ml/machine-learning-databases/00477/Real%20estate%20valuation%20data%20set.xlsx')
db1_excel.head(11)

,No,X1 transaction date,X2 house age,X3 distance to the nearest MRT station,X4 number of convenience stores,X5 latitude,X6 longitude,Y house price of unit area
0,1,2012.916667,32.0,84.87882,10,24.98298,121.54024,37.9
1,2,2012.916667,19.5,306.59470,9,24.98034,121.53951,42.2
2,3,2013.583333,13.3,561.98450,5,24.98746,121.54391,47.3
3,4,2013.500000,13.3,561.98450,5,24.98746,121.54391,54.8
4,5,2012.833333,5.0,390.56840,5,24.97937,121.54245,43.1
5,6,2012.666667,7.1,2175.03000,3,24.96305,121.51254,32.1
6,7,2012.666667,34.5,623.47310,7,24.97933,121.53642,40.3
7,8,2013.416667,20.3,287.60250,6,24.98042,121.54228,46.7
8,9,2013.500000,31.7,5512.03800,1,24.95095,121.48458,18.8
9,10,2013.416667,17.9,1783.18000,3,24.96731,121.51486,22.1


## Real Estate Valuation

Market historical data of real estate valuation collected from Sindian District, New Taipei City, Taiwan. Used for predicting house price per unit area based on location and property features.

## Columns

* No : row index
* X1 transaction date : transaction date as decimal year, e.g. 2012.917
* X2 house age : house age in years
* X3 distance to the nearest MRT station : distance in meters to nearest MRT station
* X4 number of convenience stores : count of nearby convenience stores
* X5 latitude : latitude in degrees
* X6 longitude : longitude in degrees
* Y house price of unit area : house price per unit area, in 10,000 New Taiwan Dollar per Ping